Makefile

```diff
@@ -41,19 +41,30 @@ $(PYTHON_DEPENDENCY_FILE): $(REQUIREMENTS_FILES)
 
 
 .PHONY: lint                                                                            ## Lints the code for all available runtimes
-lint:
-	cfn-lint -t $(SRC_FILES)
+lint: lint-templates lint-sam-apps
 
 
-.PHONY: test                                                                            ## Lints the code for all available runtimes
-test: lint
+.PHONY: lint-templates
+lint-templates: $(CFN_LINT_OUTPUT)
+$(CFN_LINT_OUTPUT): $(CFN_TEMPLATES)
+	cfn-lint -t $(CFN_TEMPLATES)
 
+.PHONY: test                                                                            ## Lints then tests code for all available runtimes
+test: lint test-sam-apps
+
+# Generic Sam Apps Target, loop through SAM_APPS calling make with stem
```

Comment:
This is as deep as I'll go with `Make` here. I don't thinks it's over-the-top. :wink:

Reply:
That should be fine, I understand what it's doing this time =)